deploy/.env.example

```diff
@@ -0,0 +1,21 @@
+WEB3_WS_ENDPOINT=ws://localhost:8546
+BEACON_CHAIN_RPC_ENDPOINT=http://localhost:4000
+ORACLE_PRIVATE_KEY=0x<private_key>
+SEND_TELEGRAM_NOTIFICATIONS=True
+NOTIFIERS_TELEGRAM_TOKEN=12345token
+NOTIFIERS_TELEGRAM_CHAT_ID=123456
+
+
+# GRAPH
+POSTGRES_DB=graph-node
+POSTGRES_USER=graph
+POSTGRES_PASSWORD=strong-password
+postgres_host=postgres
+postgres_user=graph
+postgres_pass=strong-password
+postgres_db=graph-node
+ipfs: 'ipfs:5001'
```

Comment:
why we change the format of the variable from here?

Reply:
@tsudmi this format required by third-party image such ipfs/graph-node/etc.